### PR TITLE
feat(#68): screen with timer creation implemented

### LINKED ISSUE #68: [UI] Implement screen with timer creation
Take a look at [Authorization feature](https://github.com/timemates/app/tree/dev/feature/authorization) as example. You gotta implement (or supplement existing stuff) data layer and domain layer (for your needs). For domain layer, you should also write tests.

Link to screen design – https://www.figma.com/file/VXjtiOeIZRIaVuN5SW2MiY/TimeMates-Design?node-id=251%3A419&mode=dev

Try to cooperate with team as there's posibility of code that can be duplicated over the features.

> **Note**
> Make sure that you read and understand our [Architecture approach](https://github.com/timemates/app/wiki/App-architecture).
> Feel free to open pull request as soon as possible so that I can review it and give some advices.

### PR DESCRIPTION
**Pull request by 1Rinkana**
Added TimerCreation(screen, reducer, state machine, middleware , also added a few lines to LocalStrings.
close #68

### CODE REVIEW

File: feature/timers/presentation/src/commonMain/kotlin/io/timemates/app/timers/ui/timer_creation/TimerCreationScreen.kt [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+import io.timemates.sdk.timers.types.value.TimerDescription
+import io.timemates.sdk.timers.types.value.TimerName
+import kotlinx.coroutines.channels.consumeEach
+import kotlin.time.Duration.Companion.minutes
+import kotlin.time.DurationUnit
+
+
+@Composable
+fun TimerCreationScreen(
+    stateMachine: StateMachine<State, Event,Effect>,
```
Comment (y9vad9) [inline feature/timers/presentation/src/commonMain/kotlin/io/timemates/app/timers/ui/timer_creation/TimerCreationScreen.kt:55]: formatting

**Review by y9vad9 — CHANGES_REQUESTED** [review]
one more moment: would be better to include screenshots of screen in PR description

**Comment by y9vad9** [comment]
make normal title of pull request please

**Comment by 1Rinkana** [comment]
Screenshot 2023-08-16 at 21 26 32

**Review by kotleni — APPROVED** [review]

**Review by y9vad9 — APPROVED** [review]

**Review by sliderzxc — APPROVED** [review]

**Pull request merged**